src/gerbil/runtime/error.ss

```diff
@@ -136,6 +136,8 @@ namespace: #f
       (unchecked-slot-set! self 'message message)
       (apply class-instance-init! self rest))))
 
+(def dump-stack-trace? (make-parameter #t))
```

Comment:
this seems _completely_ unnecessary.

Why not raise an Exception that doesn't mixin StackTrace if that's your intention?

Reply:
Because sometimes you just want the message and not the StackTrace, and it's not the business of the error object to know that—its business is to carry the information and less the handler sort it out.

Depending on who the end-user is in which context, I want to display the information or not. Maybe I want to scrub the stacktrace before I display it, or to log it in a special place and only give the user a reference.

Reply:
ok.